frontend/src/metabase/query_builder/components/NativeQueryEditor/CodeMirrorEditor/CodeMirrorEditor.module.css

```diff
@@ -1,11 +1,58 @@
 .editor {
+  --change: var(--mb-color-success);
+  --change-light: color-mix(in srgb, var(--change) 11%, transparent);
```

Comment:
Not a blocker, but I think we want to start moving away from using `color-mix` in modules, and use semantic keys for things (like mb-color-success-light). In this case it's likely fine, but it is easier to manage dark mode colors that way